src/openforms/utils/sanitizer.py

```diff
@@ -0,0 +1,190 @@
+    "feMergeNode",
+    "feMorphology",
+    "feOffset",
+    "fePointLight",
+    "feSpecularLighting",
+    "feSpotLight",
+    "feTile",
+    "feTurbulence",
+    "filter",
+    "foreignObject",
+    "g",
+    "image",
+    "line",
+    "linearGradient",
+    "marker",
+    "mask",
+    "metadata",
+    "mpath",
+    "path",
+    "pattern",
+    "polygon",
+    "polyline",
+    "radialGradient",
+    "rect",
+    "set",
+    "stop",
+    "style",
+    "svg",
+    "symbol",
+    "text",
+    "textPath",
+    "title",
+    "tspan",
+    "use",
+    "view",
+    # --- Not allowing 'a', 'animate*' and 'script' tags
+)
+
+ALLOWED_SVG_ATTRIBUTES = {
+    "*": [
+        # --- Basic presentation attributes
+        "alignment-baseline",
+        "baseline-shift",
+        "clip",
+        "clip-path",
+        "clip-rule",
+        "color",
+        "color-interpolation",
+        "color-interpolation-filters",
+        "cursor",
+        "cx",
+        "cy",
+        "d",
+        "direction",
+        "display",
+        "dominant-baseline",
+        "fill",
+        "fill-opacity",
+        "fill-rule",
+        "filter",
+        "flood-color",
+        "flood-opacity",
+        "font-family",
+        "font-size",
+        "font-size-adjust",
+        "font-stretch",
+        "font-style",
+        "font-variant",
+        "font-weight",
+        "glyph-orientation-horizontal",
+        "glyph-orientation-vertical",
+        "height",
+        "image-rendering",
+        "letter-spacing",
+        "lighting-color",
+        "marker-end",
+        "marker-mid",
+        "marker-start",
+        "mask",
+        "mask-type",
+        "opacity",
+        "overflow",
+        "pointer-events",
+        "r",
+        "rx",
+        "ry",
+        "shape-rendering",
+        "stop-color",
+        "stop-opacity",
+        "stroke",
+        "stroke-dasharray",
+        "stroke-dashoffset",
+        "stroke-linecap",
+        "stroke-linejoin",
+        "stroke-miterlimit",
+        "stroke-opacity",
+        "stroke-width",
+        "text-anchor",
+        "text-decoration",
+        "text-overflow",
+        "text-rendering",
+        "transform",
+        "transform-origin",
+        "unicode-bidi",
+        "vector-effect",
+        "visibility",
+        "white-space",
+        "width",
+        "word-spacing",
+        "writing-mode",
+        "x",
+        "y",
+        # --- Filter attributes
+        "amplitude",
+        "exponent",
+        "intercept",
+        "offset",
+        "slope",
+        "tableValues",
+        "type",
+        # --- Not allowing 'href', 'data-*', Animation and some other attributes
+    ],
+    "svg": ["xmlns", "viewBox"],
+}
+
+
+def sanitize_svg_file(data: File) -> File:
+    """
+    Function for sanitizing SVG file content.
+    The entire file content will be replaced with a sanitized version. All tags and
+    attributes that aren't explicitly allowed, are removed from the SVG content.
+    """
```

Comment:
```suggestion
def sanitize_svg_file(data: File) -> File:
    """
    Defuse an uploaded SVG file.
    
    The entire file content will be replaced with a sanitized version. All tags and
    attributes that aren't explicitly allowed, are removed from the SVG content.
    
    :arg data: the uploaded SVG file, opened in binary mode.
    """
```
there's a whole specification for how to write python docstrings: https://peps.python.org/pep-0257/

Reply:
Ahh cool 👍